src/main/resources/static/swagger.yaml

```diff
@@ -249,7 +249,7 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/HealthInformation'
-  /patient/health-information/fetch/:
+  /v1/cm/patient/health-information/fetch/:
```

Comment:
do we need /v1/cm/ prefix? why not just /v1/patient/

Reply:
We are using same for all other CM App facing APIs. We can omit the `/cm`